centreon/www/install/php/Update-25.03.0.php

```diff
@@ -17,3 +17,75 @@
  * For more information : contact@centreon.com
  *
  */
+require_once __DIR__ . '/../../../bootstrap.php';
+require_once __DIR__ . '/../../class/centreonLog.class.php';
+
+$centreonLog = CentreonLog::create();
+
+$versionOfTheUpgrade = 'UPGRADE - 25.03.0: ';
+$errorMessage = '';
+
+// -------------------------------------------- Additional Configuration -------------------------------------------- //
+/**
+ * @param CentreonDB $pearDB
+ *
+ * @throws CentreonDbException
+ *
+ * @return void
+ */
+$addConnectorToTopology = function (CentreonDB $pearDB) use (&$errorMessage): void {
+    $errorMessage = 'Unable to add insert into topology';
+    $pearDB->executeQuery(
+        <<<'SQL'
+            INSERT INTO `topology` (
```

Comment:
This script must be indempotent, you should check that the row doesn't already exist in the DB before adding it